packages/greencheck/composer.json

```diff
@@ -1,29 +1,40 @@
 {
-    "name": "thegreenwebfoundation/greencheck",
-    "type": "library",
-    "version":"1.0",
-    "license": "Apache-2.0",
-    "description": "The Green Web Foundation Greencheck package",
-    "homepage": "http://www.thegreenwebfoundation.org",
-    "authors": [
-        {"name": "Arend-Jan Tetteroo", "email": "aj@arendjantetteroo.nl"}
-    ],
-    "require": {
-        "php": ">=5.3.3",
-        "zendframework/zend-validator": "~2.0",
-        "doctrine/orm": "~2.3",
-        "symfony/validator": "~2.3|3.*|4.*",
-        "symfony/yaml": "~2.3|3.*|4.*",
-        "gedmo/doctrine-extensions": "~2.3"
-    },
-    "require-dev": {
-        "phpunit/phpunit": "3.7.*",
-        "doctrine/data-fixtures": "dev-master",
-        "friendsofphp/php-cs-fixer": "^2.14",
-        "symfony/console": "^4.2"
-    },
-    "minimum-stability" : "stable",
-    "autoload": {
-        "psr-4": { "TGWF\\Greencheck\\": "src/" }
+  "name": "thegreenwebfoundation/greencheck",
+  "type": "library",
+  "version": "1.0",
+  "license": "Apache-2.0",
+  "description": "The Green Web Foundation Greencheck package",
+  "homepage": "http://www.thegreenwebfoundation.org",
+  "authors": [
+    {
+      "name": "Arend-Jan Tetteroo",
+      "email": "aj@arendjantetteroo.nl"
     }
+  ],
+  "require": {
+    "php": ">=5.3.3",
+    "zendframework/zend-validator": "~2.0",
+    "doctrine/orm": "~2.3",
+    "symfony/validator": "~2.3|3.*|4.*",
+    "symfony/yaml": "~2.3|3.*|4.*",
+    "gedmo/doctrine-extensions": "~2.3",
+    "predis/predis": "*",
+    "symfony/polyfill-php73": "^1.11"
+  },
+  "require-dev": {
+    "phpunit/phpunit": "^8.0",
+    "doctrine/data-fixtures": "dev-master",
+    "friendsofphp/php-cs-fixer": "^2.14",
+    "symfony/phpunit-bridge": "*",
```

Comment:
i think this one and the php-mock one can now be removed again as we don't use it?